src/models/user.js

```diff
@@ -4,6 +4,7 @@ export default {
   state: {
     favoriteControllers: [],
     favoriteResults: [],
+    seenControllers: [],
```

Comment:
Are we tracking controllers the user has seen or results? In the Overview page we specify "New and unmanaged Runs" and "Saved Runs".

Reply:
@gurbirkalsi we are tracking the results which the user has already seen/opened here. I changed the naming to `seenResults` mow.